<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DwinTest</class>
 <widget class="QMainWindow" name="DwinTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DwinTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="modifyDataGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>修改数据</string>
    </property>
    <widget class="QPushButton" name="writeValueBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>写入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>指令：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="writeValueLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="variableComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="instructComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>写数据</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择U/I：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0-9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>写入数值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>选择a-z：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="select_A_Z_ComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>选择r-a-f：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="select_R_M_F_ComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>rms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fre</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>切换页面</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>目标页面：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetPageIdLineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="switchPageIdBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>切换</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="openComPortBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已发送数据：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sendTextBrowser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收到的数据：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="receivedTextBrowser">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>256</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeComPortBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
